src/main/java/http/response/ResponseFormatter.java

```diff
@@ -1,4 +1,29 @@
 package http.response;
 
+import java.io.OutputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.Map;
+
 public class ResponseFormatter {
+    private int statusCode;
+    private String statusMessage;
+    private Map<String, String> headers;
+    private byte[] body;
+
+    public void ResponseFormatter( int statusCode, String statusMessage, HashMap<String, String> headers, byte[] body) {
+        this.statusCode = statusCode;
+        this.statusMessage = statusMessage;
+        this.headers = headers;
+        this.body = body;
+    }
+
+    public String buildResponse (){
```

Comment:
Has this method been tested?

Reply:
it's old code that needs to be deleted, i think i deleted the test files fo it.